.npmignore

```diff
@@ -7,7 +7,6 @@ assets/
 build/
 coverage/
 
-lighthouse-core/scripts/*
```

Comment:
below a bit is this line:

```
!lighthouse-core/scripts/manual-chrome-launcher.js
```

can you add a direct refrence to the download-chrome sh right after it?

i'd rather do that than allowlist this whole folder.

Reply:
That sounds good, thanks. Please verify.